packages/styled-docs/pages/fade.mdx

```diff
@@ -0,0 +1,48 @@
+# Fade
+
+`Fade` in from transparent to opaque.
```

Comment:
This line is simply a sentence, so we don't need to wrap **Fade** with `` to highlight a keyword.